content/pt/docs/concepts/cluster-administration/networking.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Conectividade do Cluster
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+Conectividade é uma parte central do Kubernetes, mas pode ser desafiador 
+entender exatamente como é o seu funcionamento esperado. Existem 4 problemas
+distintos em conectividade que devem ser tratados:
+
+1. Comunicações contêiner-para-contêiner altamente acopladas: Isso é resolvido
+   por {{< glossary_tooltip text="Pods" term_id="pod" >}} e comunicações através do `localhost`.
+2. Comunicações pod-para-pod: Esse é o foco primário desse documento.
+3. Comunicações pod-para-serviço (_service_): Isso é tratado em [services](/docs/concepts/services-networking/service/).
+4. Comunicações Externas-para-serviços: Isso é tratado em [services](/docs/concepts/services-networking/service/).
```

Comment:
```suggestion
3. Comunicações pod-para-serviço (_service_): Isso é tratado em [Services](/docs/concepts/services-networking/service/).
￼4. Comunicações Externas-para-serviços: Isso é tratado em [Services](/docs/concepts/services-networking/service/).
```

Reply:
@sftim this commit appears to have something on the beginning of the line 16, otherwise I'll commit this :)

Reply:
Not sure what's wrong, but you can see the change I'm suggesting right?

Reply:
yes! I just wanted to commit directly, but will add manually here :D